### DISCUSSION TITLE
Docker container including MBDyn and MFront

### DISCUSSION THREAD

**Discussion by octave-user** [thelfer/MFrontGenericInterfaceSupport#136] [General]
Dear Mr. Thomas Helfer,

I have set up an automatic build job, hosted at github.com, in order to publish docker images including always the latest stable release of GNU-Octave, MBDyn and also MFrontGenericInterfaceSupport plus MFrontGallery.

- [github.com](https://github.com/octave-user/mboct-fem-pkg)
- [hub.docker.com](https://hub.docker.com/repository/docker/octaveuser/mboct-fem-pkg/general)

The only thing I would like to ask you is, to have a look at the image and maybe to give a few suggestions. Furthermore it would be nice if you could provide a functional "make package" command in your CMakeLists.txt. Actually it is working well with tfel but not with MGIS and also not with MFrontGallery.

Reinhard

**Comment by octave-user** [comment]
Regarding `make package` and `make package_source`, see also the related pull requests:
- https://github.com/thelfer/tfel/pull/608
- https://github.com/thelfer/MFrontGenericInterfaceSupport/pull/137
- https://github.com/thelfer/MFrontGallery/pull/57

**Reply by thelfer** [reply]
Hi @octave-user,
I am currently on vacations, so no quite responsive. I'll try to have a look ASAP, but this looks really cool.
Best,
Thomas
